inc/simplepdf.class.php

```diff
@@ -395,6 +383,48 @@ public function displaySpace($nb = 1)
         $this->pdf->Ln(4 * $nb);
     }
 
+    /**
+     * Clean table HTML for proper PDF rendering
+     * Removes problematic attributes and styles that cause tables to overflow
+     *
+     * @param string $html Table HTML code
+     * @return string Cleaned HTML
+     */
+    private function cleanTableHtml($html)
+    {
+        // Remove colgroup entirely (causes fixed widths)
+        $html = preg_replace('/<colgroup\b[^>]*>.*?<\/colgroup>/is', '', $html);
+
+        // Remove table-layout:fixed style (prevents auto-sizing)
+        $html = preg_replace('/table-layout\s*:\s*fixed\s*;?/i', '', $html);
+
+        // Remove ALL width styles from elements (including px, %, em, etc.)
+        $html = preg_replace('/\bwidth\s*:\s*[^;"\'>]+;?/i', '', $html);
+
+        // Remove ALL height styles from elements
+        $html = preg_replace('/\bheight\s*:\s*[^;"\'>]+;?/i', '', $html);
+
+        // Remove width attributes from any tag
+        $html = preg_replace('/(<[^>]+)\s+width\s*=\s*["\']?[^"\'\s>]+["\']?/i', '$1', $html);
+
+        // Remove height attributes from any tag
+        $html = preg_replace('/(<[^>]+)\s+height\s*=\s*["\']?[^"\'\s>]+["\']?/i', '$1', $html);
```

Comment:
For tables, it’s best to limit changes to what is strictly necessary and to make targeted modifications, even if it means adding further adjustments later. For instance, downsized images might lose their formatting if all size properties are removed.